copilot/fsd-frontend/manifest.yml

```diff
@@ -0,0 +1,84 @@
+# The manifest for the "fsd-assessment" service.
+# Read the full specification for the "Load Balanced Web Service" type at:
+#  https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/
+
+# Your service name will be used in naming your resources like log groups, ECS services, etc.
+name: fsd-frontend
+type: Load Balanced Web Service
+
+# Distribute traffic to your service.
+http:
+  # Requests to this path will be forwarded to your service.
+  # To match all requests you can use the "/" path.
+  path: '/'
+  # You can specify a custom health check path. The default is "/".
+  healthcheck: '/healthcheck'
+  alias: frontend.test.access-funding.test.levellingup.gov.uk
+
+# Configuration for your containers and service.
+image:
+  # Docker build arguments. For additional overrides: https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#image-location
+  location: ghcr.io/communitiesuk/funding-service-design-frontend:latest
+  # Port exposed through your container to route traffic to it.
+  port: 8080
+
+# Valid values: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html
+# Number of CPU units for the task.
+cpu: 256
+# Amount of memory in MiB used by the task.
+memory: 1024
+
+# See https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#platform
+platform: linux/x86_64
+# Number of tasks that should be running in your service.
+count: 1
+# Enable running commands in your container.
+exec: true
+
+network:
+  connect: true # Enable Service Connect for intra-environment traffic between services.
+
+# storage:
+  # readonly_fs: true       # Limit to read-only access to mounted root filesystems.
+
+# Optional fields for more advanced use-cases.
+#
+# Pass environment variables as key value pairs.
+variables:
+  # SENTRY_DSN: "https://5ea1346b1c1b4dd8af0c95435ca77945@o1432034.ingest.sentry.io/4503903486148608"
+  FLASK_ENV: ${COPILOT_ENVIRONMENT_NAME}
+  COOKIE_DOMAIN: ".test.levellingup.gov.uk"
+  APPLICANT_FRONTEND_HOST: "https://frontend.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk"
```

Comment:
When we get to prod, we'll have to have an option to remove the .test - not relevant for now - just a comment.

Reply:
https://github.com/communitiesuk/funding-service-design-frontend/pull/394/files/13d9c8f51e53ab7a05b6f266b7a3004776626fcc#diff-e23689f3156b83727ff59dcfa44433b3bf8897721b122e9f6df93df2f97707a2R73-R80 Already been considered :)